### PR TITLE
[12.0][IMP] sale_order_margin_percent: Add margin percent field in sales order line

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT31041

ping @chienandalu @carlosdauden

### CODE REVIEW

File: sale_order_margin_percent/models/sale_order_margin.py [hunk 1]
```diff
@@ -17,3 +17,19 @@ def _compute_percent(self):
         for order in self:
             if order.margin and order.amount_untaxed:
                 order.percent = (order.margin / order.amount_untaxed) * 100
+
+
+class SaleOrderLine(models.Model):
+    _inherit = 'sale.order.line'
+
+    percent = fields.Float(
```
Comment (pedrobaeza) [inline sale_order_margin_percent/models/sale_order_margin.py]:
```suggestion
    margin_percent = fields.Float(
```

Comment (sergio-teruel) [reply sale_order_margin_percent/models/sale_order_margin.py]: Changes done"!

File: sale_order_margin_percent/models/sale_order_margin.py [hunk 2]
```diff
@@ -17,3 +17,20 @@ def _compute_percent(self):
...
+
+
+class SaleOrderLine(models.Model):
+    _inherit = 'sale.order.line'
+
+    margin_percent = fields.Float(
+        string='Margin(%)',
+        compute='_compute_margin_percent',
+        digits=(16, 2),
+        )
```
Comment (pedrobaeza) [inline sale_order_margin_percent/models/sale_order_margin.py]: Is this indentation accepted by the linter?

**Review by pedrobaeza — APPROVED** [review]

**Comment by sergio-teruel** [comment]
fixed

**Review by carlosdauden — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]

**Pull request merged**